src/promptflow-evals/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 Please insert change log into "Next Release" ONLY.
 
 ## Next release
+- Exposed batch evaluation run timeout variable via "PF_BATCH_TIMEOUT_SEC" environment variable

```

Comment:
Let's use the following CHANGELOG to align with the promptflow CHANGELOG format.

```
# Release History

## v0.3.2 (Upcoming)

### Features Added

### Bugs Fixed

### Improvements
- Exposed batch evaluation run timeout variable via "PF_BATCH_TIMEOUT_SEC" environment variable

## v0.3.1 (2022-07-09)
- This release contains minor bug fixes and improvements.

## v0.3.0 (2024-05-17)
- Initial release of promptflow-evals package.
```

Reply:
done